apps/SageAccountWeb/src/components/SageResourcesPage.tsx

```diff
@@ -24,23 +22,18 @@ export const SageResourcesPage = (props: SageResourcesPageProps) => {
           width: '900px',
         }}
       >
-        <IconButton
-          onClick={() => {
-            history.goBack()
-          }}
-          sx={{
-            position: 'absolute',
-            top: theme.spacing(1.5),
-            left: theme.spacing(1.5),
-          }}
-        >
-          <ArrowBackIcon />
-        </IconButton>
         <Box
           sx={{
-            padding: '40px',
+            px: '64px',
+            paddingTop: '64px',
```

Comment:
Adjust padding to better match [Figma](https://www.figma.com/file/TOcmJ2QHr51v8Huv3wIec9/Sage-Registration-App?node-id=1166-33862&t=Bg7ymOI4wD2nsmHv-4) and so back button positioning matches other locations